extended-app-support.html.md.erb

```diff
@@ -23,14 +23,19 @@ To enable cflinuxfs3 support in <%= vars.app_runtime_abbr %> 10.2.1, operators m
     * **Errands:** Configure Post-Deploy and Pre-Delete errands related to the buildpacks and stack.
     * **Resource Config:** Select the VM type to use for executing the errand.
 
-1. Click **Apply Changes** to roll out the Extended App Support tile.
+1. **Apply Changes** to roll out the Extended App Support tile.
```

Comment:
@In Ops Manager, "Apply Changes" is a button. Why did you remove the word "Click"?